packages/loader/container-loader/src/location-redirection-utilities/index.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export {
```

Comment:
There is no need of 2 index.ts files. Just export this from the outside main one.

Reply:
Drilling into directories is generally considered bad practice, and our linter rules explicitly forbid it outside of particular contexts.